Modules/OrgUnit/classes/LocalUser/class.ilLocalUserGUI.php

```diff
@@ -57,13 +59,19 @@ public function __construct(ilObjectGUI $parentGui)
         $this->access = $DIC->access();
         $this->tabsGui = $DIC->tabs();
         $this->logger = $DIC->logger();
+        $this->ui_factory = $DIC['ui.factory'];
 
         $this->lng->loadLanguageModule('user');
         if (!$this->rbacSystem->checkAccess("cat_administrate_users", $this->parentGui->getObject()->getRefId())) {
             $this->tpl->setOnScreenMessage('failure', $this->lng->txt("msg_no_perm_admin_users"), true);
         }
     }
 
+    public function getRefId()
+    {
+        return $_GET['ref_id'];
```

Comment:
Please use the [request wrapper](https://github.com/ILIAS-eLearning/ILIAS/blob/release_8/src/HTTP/Wrapper/WrapperFactory.php#LL38C22-L38C22) here.